widgetssdk/src/main/java/com/glia/widgets/view/OperatorStatusView.java

```diff
@@ -151,6 +166,29 @@ public void setShowRippleAnimation(boolean show) {
         }
     }
 
+    private void updatePlaceholderView(
+            int size,
+            int contentPadding,
+            int visibility
+    ) {
+        placeholderView.post(() -> {
```

Comment:
why using post if all the invokers of this method manipulate ui without post?

Reply:
this is related to .setConentPadding which is not currently working properly.

see:
https://github.com/material-components/material-components-android/issues/2063

it might get resolved in future releases.
I can extract other method calls from this post method and leave only content padding part for now with comment related to this issue

what do you think? @jakob-mc

Reply:
sounds good. + add a comment.